base_control/BNO.h

```diff
@@ -7,6 +7,7 @@
 #include <Adafruit_BNO055.h>
 #include <utility/imumaths.h>
 
+#include <stdint.h>

```

Comment:
i thought it was needed because of the using of uint8_t in the orientationStatus function but in the bno defualt headers is already included. I removed it in a new commit.